app/controllers/api/schools_controller.rb

```diff
@@ -5,7 +5,7 @@ class SchoolsController < Api::ApplicationController
     def index
       return error_response if invalid_query?
 
-      @schools = SchoolSearch.call(args)
+      @schools = SchoolSearch.call(args).to_h[:schools]
 
       render json: { schools: @schools.as_json(only: %i[id name urn town postcode]) }
```

Comment:
Looks good, however, for the sake of clarity can I recommend slightly different naming?
```suggestion
      @school_search = SchoolSearch.call(args).to_h[:schools]

      render json: { schools: @school_search.schools.as_json(only: %i[id name urn town postcode]) }
```